hoomd/operation.py

```diff
@@ -80,15 +79,7 @@ def _setattr_hook(self, attr, value):
 
     def _setattr_param(self, attr, value):
         """Hook for setting an attribute in `_param_dict`."""
-        old_value = self._param_dict[attr]
         self._param_dict[attr] = value
-        new_value = self._param_dict[attr]
-        if self._attached:
-            try:
-                setattr(self._cpp_obj, attr, new_value)
```

Comment:
With this removal, how to parameters get set on C++ objects?

Reply:
`ParameterDict` has already handled this directly since #1178. This is removing (for the most part) unused code.